cellcounter/cc_kapi/factories.py

```diff
@@ -21,14 +21,48 @@ class Meta:
     is_active = True
     is_superuser = False
 
+class DefaultKeyboardsFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = DefaultKeyboards
+
+
+class DefaultKeyboardFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = Keyboard
+        strategy = factory.BUILD_STRATEGY
+
+    class Params:
+        mappings = None
+
+    id = 0
+    user = None
+
+    #@factory.post_generation
```

Comment:
Fixed in https://github.com/cellcounter/cellcounter/pull/446/commits/992d7b78de98daf8f018a2bac88c3b31e0337eaf